### DISCUSSION TITLE
Eruption stops after waking from suspend

### DISCUSSION THREAD

**Discussion by blazblu82** [X3n0m0rph59/eruption#199] [General]
I've been using your software for a while now. The only issue I'm having, aside from experimental support for my Vulcan TKL and Kone XP, is the main service stops when waking from sleep and I have to reactive via console each time. Is there a way to allow it to persist? System restarts don't affect it. I'm running Fedora 37 + KDE. Thanks!

**Comment by X3n0m0rph59** [comment]
Hi @blazblu82, I recently fixed multiple bugs regarding the resume from suspend functionality and device hotplug as well. The code has been pushed to the `develop` branch. Chances are that it may be working soon please stay tuned...
